kolibri/plugins/user/assets/src/views/SignUpPage.vue

```diff
@@ -322,7 +311,11 @@
           'It will be visible to administrators. It will also be used to help improve the software and resources for different learner types and needs.',
         context: '\nDetails on how the demographic information requested in the form will be used.',
       },
-      accessAsGuest: 'Explore without account',
+      signInPrompt: {
```

Comment:
You'll need to use the `crossComponentTranslator` to "borrow" this message from `AuthSelect`.

(Heads up that there might be a problem if this causes a circular dependency though, e.g. `AuthSelect > SignUpPage > crossComponentTranslator(AuthSelect)`)

Reply:
@jonboiser thanks for the tip. I didn't know this.
I have done it and checked it works correctly:
![image](https://user-images.githubusercontent.com/1008178/86626676-4f6a8700-bfc7-11ea-9a96-a88dac026e25.png)

Reply:
@radinamatic  btw, the translation to Spanish is not correct, it says "_..tienea una ..._" , it should be "_... tiene una..._". Also there's not concordance in verbal times. I'd propose to change it to "_Inicie sesión si ya tiene una cuenta_"